actors/builtin/market/market_test.go

```diff
@@ -2755,17 +2804,32 @@ func (h *marketActorTestHarness) generateDealAndAddFunds(rt *mock.Runtime, clien
 	return deal4
 }
 
-func generateDealProposal(client, provider address.Address, startEpoch, endEpoch abi.ChainEpoch) market.DealProposal {
+func (h *marketActorTestHarness) generateDealWithCollateralAndAddFunds(rt *mock.Runtime, client address.Address,
+	minerAddrs *minerAddrs, providerCollateral, clientCollateral abi.TokenAmount, startEpoch, endEpoch abi.ChainEpoch) market.DealProposal {
+	deal4 := generateDealProposalWithCollateral(client, minerAddrs.provider, providerCollateral, clientCollateral,
```

Comment:
What happened to deal1, deal2, and deal3?

Reply:
heh yeah I had the same question.  I was propagating the existing convetion, will just change to deal.